client/my-sites/plans/ecommerce-trial/upgrade-confirmation/confirmation-task/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { Card } from '@automattic/components';
+
+import './style.scss';
+
+interface ConfirmationTaskProps {
+	title: string;
+	subtitle: string;
```

Comment:
If we want to get super fancy, we could type these as `TranslateResult` and not just `string`. But that's not blocking. (The main gotcha is that `translate()` returns a `TranslateResult` as the content may include some HTML tags - to work around that you need to pass in an option to type the return as a string.)
```suggestion
	title: TranslateResult;
	subtitle: TranslateResult;
```